src/types/types.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Todo } from './Todo';
+
+export type CreateTodoArgs = Omit<Todo, 'id'>;
+export type UpdateTodoData = Partial<Omit<Todo, 'id' | 'userId'>>;
+
+export enum Filter {
+  All = '',
+  Active = 'Active',
+  Completed = 'Completed',
```

Comment:
enum naming convention
```suggestion
  ALL = '',
  ACTIVE = 'Active',
  COMPLETED = 'Completed',
```